### DISCUSSION TITLE
Bootstrap 4 & 5 + Vuejs

### DISCUSSION THREAD

**Discussion by modris-it** [bootscore/bootscore#292] [Q&A] [answered]
Hi! 
How to integrate **Vue js** with **Bootscore** templates 4 and 5?

**Comment by justinkruit** [comment, marked as answer]
You don't, because VueJS is a Javascript Framework and Bootscore is a WordPress theme.

**Reply by justinkruit** [reply]
To clarify, with VueJS you'd make an application that is completely javascript based. Bootscore is a WordPress theme, and WordPress won't work with just a JS framework. What you can do is make a VueJS application which connects to WordPress, but then WordPress would work as a headless CMS removing the possibility to use WP themes.
